examples/minimal/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@example/minimal",
+  "version": "0.0.1",
+  "type": "module",
```

Comment:
Personally love this, but concerned that it might trip up users that are unfamiliar with the implications of this. What do you think?

Reply:
I thought about omitting `"type": "module"`, too, so once again I’m so glad you asked about it.

I added it because I preferred the trade-offs. Without it, users are currently instructed to use [`astro.config`**`.mjs`**](https://docs.astro.build/reference/configuration-reference) and [`snowpack.config`**`.js`**](https://www.snowpack.dev/reference/configuration). That tripped me up.

The benefit of nothing or `"type": "common"` historical CommonJS-Node behavior. However, historical CommonJS-Node behavior is only half the way Node works. Also, Node is only 1/3 the environment for developing JS, considering Deno and web browsers themselves. Finally, `.astro` files themselves do not encourage CommonJS-Node behavior. The benefit of `"type": "module"` is consistency, as modules are the most interoperable space between them. 

I’ll fully respect the wishes of the team, of course, but my opinion is to _keep it_. And maybe even consider it for some of the other templates, too. 😉

Reply:
This is also a change I 100% agree with! But for now, let's remove it and use `astro.config.mjs`.

I just want to give us some time to:
1. update the documentation to use `astro.config.js`
2. update the other templates to use `"type": "module"` 

Otherwise this template has the potential to be confusing because it's out of sync with the other ones.

Reply:
I respect that. Thanks for expressing support while explaining why you’d like it to work differently for now. Yay OSS. Yay team work makes the dream work!

Done in a5d39ea.